src/transformer/decode.ts

```diff
@@ -802,6 +833,21 @@ export class Decoder {
     )
   }
 
+  public buildKeywordsContents = (manuscriptKeywords: ManuscriptKeyword[]) => {
+    const p = document.createElement('p')
```

Comment:
I think there was a keywords_element node already created to act as a wrapper for those keywords elements. It should be generated using a node from the PM schema. It is probably important to note that Decoder doesn't build a browser/html DOM. It builds a ProseMirror DOM (exists only in memory) and later when we create a view, only then Prosemirror builds an html view for the nodes in the DOM tree, that we described in the schema/node

Reply:
Fixed.